handlers/logentries/logentries.go

```diff
@@ -0,0 +1,37 @@
+// Package LogEntries implements a LogEntries JSON handler.
+package logentries
+
+import (
+	"encoding/json"
+
+	"github.com/apex/log"
+	"github.com/bsphere/le_go"
+)
+
+type (
+	LogHandler struct {
+		logger *le_go.Logger
+	}
+)
+
+func New(token string) (*LogHandler, error) {
+	logger, err := le_go.Connect(token)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return &LogHandler{
```

Comment:
Wasn't 100% sure here if I could nix the previous guard and `return &LogHandler{logger: logger}, err` here